tools/logger/convert.c

```diff
@@ -268,7 +274,12 @@ static void process_params(struct proc_ldc_entry *pe,
 			/* %pQ format specifier */
 			/* substitute log entry address with formatted entry text */
 			pe->params[i] = (uintptr_t)asprintf_entry_text(raw_param);
-			pe->subst_mask |= 1 << i;
+
+			if (pe->params[i])
+				pe->subst_mask |= 1 << i;
+			else
+				pe->params[i] = (uintptr_t)missing;
+
```

Comment:
This branch is now only used for one trace in SOF:

```
./trace/trace.c:139:                 _TRACE_INV_ID, _TRACE_INV_ID, "Suppressed %u similar messages: %pQ",
```

But still, for correctness, the entry may not be found.